### DISCUSSION TITLE
[Enhancement] [select-v2] selectv2选择器支持自定义label字段

### DISCUSSION THREAD

**Discussion by mswedgar** [element-plus/element-plus#13290] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select-v2`

## Description

目前selectv2组件仅支持 label字段作为option的展示值，希望能指定label字段例如 
```vue
<el-select-v2 value-key="key" label-key="name"></el-select-v2>
```

### Additional comments

_(empty)_

**Comment by kooriookami** [comment]
https://element-plus.org/zh-CN/component/select-v2.html#%E8%87%AA%E5%AE%9A%E4%B9%89%E9%80%89%E9%A1%B9
此功能已存在

**Reply by DusuWen** [reply]
但是这个功能需要启用multiple
